### ISSUE TITLE
Negative loss

### ISSUE THREAD

**Issue by nishavarghese15** [nianticlabs/monodepth2#447]

**Comment by nishavarghese15** [comment]
Hi,

Thanks for your excellent work. I am trying to train the network using a custom dataset. Is there any chance of getting 'negative losses' during training?
![image](https://user-images.githubusercontent.com/93310210/181717537-e9bc1832-9bc3-4e1a-8960-b13919b448fd.png)

Thanks in advance!

**Comment by zchoi** [comment]
Have you solved this problem? I met the same issue.

**Comment by nishavarghese15** [comment]
> Have you solved this problem? I met the same issue.

Hi, I solved it. But I dont remember exactly what I did. But for a custom dataset, i have changed —max_depth, —min_depth options. Then I remember I had to use dataset with sufficient number of images. Initially I thought for a small number of images (around 200) it will overfit, but it didnt happen. I had to increase dataset size to thousands of images (>5000) for training. From my experience, this code will work for any dataset.
